stdlib/Random/docs/src/index.md

```diff
@@ -18,9 +18,7 @@ The PRNGs (pseudorandom number generators) exported by the `Random` package are:
 Most functions related to random generation accept an optional `AbstractRNG` object as first argument.
 Some also accept dimension specifications `dims...` (which can also be given as a tuple) to generate
 arrays of random values.
-In a multi-threaded program, you should generally use different RNG objects from different threads
-or tasks in order to be thread-safe. However, the default RNG is thread-safe as of Julia 1.3
-(using a per-thread RNG up to version 1.6, and per-task thereafter).
+In a multi-threaded program, you should generally use different RNG objects from different threads or tasks in order to be thread-safe and deterministic (if given a specific seed). However, the default global RNG is thread-safe as of Julia 1.3 (using a per-thread RNG up to version 1.6, and per-task thereafter). The global RNG refers to per-thread RNG instances; If you instantiate a non-global RNG, you should create a new RNG instance for each thread to avoid possible data-races.
```

Comment:
The

> The global RNG refers to per-thread RNG instances; If you instantiate a non-global RNG, you should create a new RNG instance for each thread to avoid possible data-races.

seems like it is just saying the same thing as the earlier stuff in the paragraph again?

Reply:
Here was my confusion so maybe there's a clearer way to get this across. Per the prior docs:
1. The global RNG is thread safe
2. Xoshiro is used as the global RNG 
3. Therefore, if I use a newly instantiated Xoshiro (call it `X`), I can have code in multiple threads reference `X` without issues.

#3 is not correct because the Global RNG is constructed in a way that makes it thread safe and that's the point that I don't think comes across previously and what 

>The global RNG refers to per-thread RNG instances; If you instantiate a non-global RNG, you should create a new RNG instance for each thread to avoid possible data-races.

Is trying to differentiate from what was stated before.

Reply:
I tried to clarify in the latest commit